.circleci/config.yml

```diff
@@ -337,7 +337,7 @@ jobs:
   binary_macos_wheel:
     <<: *binary_common
     macos:
-      xcode: "9.4.1"
```

Comment:
Shouldn't we set the version at one place and use that variable everywhere so that it's easier to change in this kind of scenario?

Reply:
That is a good idea. I would be in favour of merging this as-is and making the improvements on a separate PR because this one fixes part of our CI and uncovers issues with some of the unit-tests. Thoughts?

Reply:
Makes sense, I can look into doing that.